docs/manage-rabbitmq.md

```diff
@@ -0,0 +1,29 @@
+---
+title: How to Manage RabbitMQ
+---
+<!--
+Copyright (c) 2024 Broadcom. All Rights Reserved. The term "Broadcom" refers
+to Broadcom Inc. and/or its subsidiaries.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License,
+Version 2.0 (the "License”); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# How to Manage RabbitMQ
+
+This section targets **administrators** who want to operate single RabbitMQ
+instance and clusters of RabbitMQ nodes. It provides documentation for
```

Comment:
slight rewrite but only a suggestion: The section is mainly for a **administrators** who want to operate a single.....

using the word "mainly" to mean "for the most part" so to cover ourselves that all of it may not be for admins